### ISSUE TITLE
can not move mouse cursor to second monitor on top

### ISSUE THREAD

**Issue by uocxp** [mlutfy/hidetopbar#174]
first of all thanks a lot for the extension ^^. 
I was very happy using this extension on one monitor but after i connected my laptop to an external monitor on top, i am unable to navigate my mouse cursor to the second monitor. 
I will be very thankful if this issue could be resolved :)

**Comment by tuxor1337** [comment]
This is "intended". It's a pressure barrier that implements the feature "Show panel when mouse approaches edge of the screen" (see preferences). You can disable this preference.

**Comment by uocxp** [comment]
ok so i musst use a shortcut to get the topbar shown when i need to access a tray icon for example.

**Comment by tuxor1337** [comment]
That should work, yes.

Does it solve this issue for you? If not, please describe the behaviour you would prefer, then we can turn this issue into an "enhancement" request.

**Comment by uocxp** [comment]
yes it works. I would though prefer to be able to autohide the topbar using the mouse without getting blocked.

**Comment by tuxor1337** [comment]
How exactly would you want to use your mouse to "unhide"?

**Comment by kalbr** [comment]
I would expect the top bar to "unhide" when the mouse is at (or very near) the top edge of the screen for some small amount of time. This is how macOS does it. Usually I have the upper monitor offset so that part of the top edge of the bottom screen can be 'pressed against' without going on to the other monitor.

If it happens that there is no part of the edge to press against it is still possible (though annoying) to bring up top menu. I think this behaviour is better than locking out the upper monitor.

**Comment by easynowbaby** [comment]
> I would expect the top bar to "unhide" when the mouse is at (or very near) the top edge of the screen for some small amount of time. This is how macOS does it. Usually I have the upper monitor offset so that part of the top edge of the bottom screen can be 'pressed against' without going on to the other monitor.
> 
> If it happens that there is no part of the edge to press against it is still possible (though annoying) to bring up top menu. I think this behaviour is better than locking out the upper monitor.

agreed, this would be very much appreciated

**Comment by TimTomTwo** [comment]
+1 to kalbr's method.
I recently bought a secondary monitor and now I stumbled into this problem too.
I did not expect my mouse to be "trapped" in my lower (=main) monitor, just because I enabled that "unhide, when mouse pointer is nearby" option. 
I would prefer, that the Top Bar appears, when my mouse pointer is nearby, BUT it can still go past it, to the next monitor above.
This would inadvertently cause the Top Bar to appear and disappear accidentally, just because I'm switching monitors, but that would be totally fine by me.
